src/viam/sdk/registry/registry.cpp

```diff
@@ -102,7 +102,7 @@ std::shared_ptr<const ModelRegistration> Registry::lookup_model(const API& api,
                                                                 const Model& model) {
     const std::lock_guard<std::mutex> lock(lock_);
     const std::string name = api.to_string() + "/" + model.to_string();
-    return lookup_model_inlock_(std::move(name), std::move(lock));
+    return lookup_model_inlock_(name, std::move(lock));
```

Comment:
Do you need the `move` on lock either? It's const.

Reply:
Ah, nope! Surprised it let us keep that. Fixed.